README.md

```diff
@@ -43,16 +63,22 @@ cd ~/anymal_locomotion_ws/
 
 catkin config --extend ~/subt_ws/install/
 catkin config --cmake-args -DCMAKE_BUILD_TYPE=Release
-
-catkin build cerberus_anymal_b_control_1
 ```
 
-Launch the ANYmal locomotion controller:
+Build and launch the locomotion controller for ANYmal B:
 ```
+catkin build cerberus_anymal_b_control_1
```

Comment:
Basic naming question: Why are we appending the 1? Is this still needed to discriminate something?

Reply:
This means controller config 1.

Reply:
It follows the naming scheme with sensor_config_*